frontend/src/scenes/actions/EventName.js

```diff
@@ -1,20 +1,35 @@
 import React from 'react'
-import Select from 'react-select'
+import { Select } from 'antd'
 import { useValues } from 'kea'
 import { userLogic } from 'scenes/userLogic'
 
-export function EventName({ value, onChange }) {
+export function EventName({ onChange }) {
     const { eventNamesGrouped } = useValues(userLogic)
+
     return (
         <span>
             <Select
-                options={eventNamesGrouped}
-                isSearchable={true}
-                isClearable={true}
+                showSearch
```

Comment:
value prop is gone on this select so it doesn't automatically show you the current event